locale/fr/get-involved/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Get involved
+layout: contribute.hbs
+---
+
+# Nous encourageons toutes les formes de contributions de la part de la communauté 
+
+La communauté Node.js est vaste, inclusive et excitée de permettre à autant 
+d'utilisateurs de contribuer de toutes les manières possibles. Si vous souhaitez
+[reporter un problème](https://github.com/nodejs/node/issues), aider à la documentation
+ou contribuer au code du projet, vous êtes au bon endroit. Explorez les ressources 
+de notre communauté pour trouver comment vous pouvez aider:
+
+<hr>
+
+## Discussion
+
+- La [list des tickets GitHub](https://github.com/nodejs/node/issues) est le bon endroit pour discuter des fonctionnalités coeurs de Node.js.
+
+- Pour une discussion temps réelle à propos de développement Node, allez sur `irc.freenode.net` dans le canal `#node.js` avec un [client IRC](https://fr.wikipedia.org/wiki/Liste_de_clients_IRC) ou connectez vous depuis votre navigateur web sur le canal en utilisant le canal [freenode's WebChat](http://webchat.freenode.net/?channels=node.js).
```

Comment:
I think it should be "temps réel", masculine, and "*du* développement Node".
And "canal" is repeated too many times in the last part. Maybe "sur le canal en utilisant le [WebChat freenode]"?